middleware/index.js

```diff
@@ -1,14 +1,18 @@
 const express = require('express');
 const path = require('path');
+const fs = require('fs');
 
-module.exports = ({ middlewarePath = '/falcor-postman', app }) => {
+module.exports = ({ middlewarePath = '/falcor-postman', falcorModelPath = '/model.json', app, fileSystem = fs }) => {
```

Comment:
as far as I understood because of HMR the loading of the HTML file should be performed by the `webpack-dev-middleware.fileSystem` object but I might be wrong (have a look at /falcor-postman/example/webpackServeBundle.js line 26)